packages/scheduler/index.js

```diff
@@ -7,4 +7,37 @@
 
 'use strict';
 
-export * from './src/Scheduler';
+const isBrowserEnvironment =
+  typeof window !== 'undefined' && typeof MessageChannel === 'function';
```

Comment:
Could I just drop this check and only export the DOM version if I update the callers during the sync?

Reply:
The inline require breaks the build but I'll wait to hear back on this before fixing it.

Reply:
For www you can lift the checks into the outer module wrapper, but for OSS this seems like the right place to put them.

Reply:
Actually I guess the right place is here? https://github.com/facebook/react/blob/master/packages/scheduler/npm/index.js

I don't know what the best practice is in the npm ecosystem. Ideally we would handle this at the bundler level, using something like the `browser` field.

Reply:
Perfect, thanks!